test/phoenix_live_view/html_formatter_test.exs

```diff
@@ -1313,6 +1313,21 @@ if Version.match?(System.version(), ">= 1.13.0") do
       )
     end
 
+    test "force inline element into a newline when followed by a eex expression" do
+      input = """
+        <%= some_function("arg") %><span>content</span>
```

Comment:
I guess we shouldn't break line here since there isn't white space between eex and span. Otherwise it will render an unwanted space.

Reply:
Oops☝🏻  is wrong. it should remain as is. It should only break in case there is a space between eex and the inline tag.

Reply:
Sounds good, so I'm gonna update the code to not break line as the example. It does solve the case I have here.

Reply:
I mean, it is not wrong but I guess it is easier to keep as `<%= some_function("arg") %><span>content</span>`. That's what we do for instance here:

https://github.com/phoenixframework/phoenix_live_view/blob/master/test/phoenix_live_view/html_formatter_test.exs#L1256

It only breaks the inline element to the next line in case it has attributes.

Reply:
You mean it's better to `:preserve` by default and let it break into next only if attrs doesn't fit the line (which is the current behavior)?

Reply:
I don’t think we should preserve. Preserve means don’t touch anything inside. This is about how two different tags “connect”.

Reply:
Oh, ignore me. We should not change outside nor inside. Yes, preserve is the way to go.